### ISSUE TITLE
Document Translation in Korean

### ISSUE THREAD

**Issue by jeffrey-kor** [jitsi/handbook#211]
Can I translate this for Korean? :)

**Comment by saghul** [comment]
Definitely! I think Docusaurus can be configured to support multiple languages, but I have never tried it.

Do you know how to accomplish that?
